.github/workflows/chromatic.yaml

```diff
@@ -8,9 +8,6 @@ on:
   pull_request_review:
     types: [submitted]
   push:
-    paths:
```

Comment:
Remove path filter - We want it to trigger for any push and then filter later on